src/components/Onboarding/onboardingData.ts

```diff
@@ -52,4 +54,25 @@ const states = [
   { label: 'WY', value: 'WY' },
 ];
 
-export default states;
+export const homeInfo = {
```

Comment:
`homeInfo` can be typed as `Home` from `API.ts`, we can just hardcode the required filed `__typename: "Home"`

Reply:
We can't use type home because when sending __typename to the backend produces an error. I can't remove it from the JSON object before sending either since it has the __ which is considered an object member. Using homeInfo gives default values and avoids this error.

Reply:
After discussing this in the meeting, we were able to find a solution by using a different type provided by AWS.